docs/infrastructure/native-image.md

```diff
@@ -228,16 +228,41 @@ to allow use of some library functions (like `IO.println`) in the _Native Image_
 built runner.
 
 The support can be enabled by setting environment variable `ENSO_JAVA=espresso`
-and making sure Espresso is installed in GraalVM executing the Enso engine -
-e.g. by running `graalvm/bin/gu install espresso`. Then execute:
+and making sure Espresso is installed in the Enso engine `component` directory:
+
+```bash
+enso$ built-distribution/enso-engine-*/enso-*/component/
+```
+
+e.g. next to `js-language-*.jar` and other JARs. Download following these two
+JARs (tested for version 23.1.1) and copy them into the directory:
```

Comment:
Ideally there should be an sbt target for that

Reply:
Just an idea: The `espresso-*.jar` can also be distributed in the `component` directory by default. Having them there and not loading any classes from them should not have any impact on the startup or peak performance.

Reply:
But the download size... I am afraid to compromise it only for an experimental feature.